bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/scheduling/Strand.java

```diff
@@ -71,15 +70,14 @@
     public boolean blockedOnExtern;
     public Set<ChannelDetails> channelDetails;
     public Set<SchedulerItem> dependants;
-    public ObserverContext observerContext;
     public boolean cancel;
 
     SchedulerItem schedulerItem;
     List<WaitContext> waitingContexts;
     WaitContext waitContext;
     ItemGroup strandGroup;
 
-    private Map<String, Object> globalProps;
+    public Map<String, Object> globalProps;
```

Comment:
Why did we make this public? I thought we are getting rid of it?

Reply:
We will hide this from others using java 9 modules. but needs to be public here because I need to access it from Environment.

Reply:
fill fix